pkgs/top-level/perl-packages.nix

```diff
@@ -3437,6 +3437,13 @@ let
       url = mirror://cpan/authors/id/A/AJ/AJGB/Crypt-Curve25519-0.06.tar.gz;
       sha256 = "1ir0gfxm8i7r9zyfs2zvil5jgwirl7j6cb9cm1p2kjpfnhyp0j4z";
     };
+    patches = [
+      (fetchpatch {
+        url = "https://gitweb.gentoo.org/repo/gentoo.git/plain/dev-perl/Crypt-Curve25519/files/Crypt-Curve25519-0.60.0-fmul-fixedvar.patch";
+        name = "Crypt-Curve25519-0.60.0-fmul-fixedvar.patch";
```

Comment:
I think it should work without `name`
`name` is inevitable when filename in `url` contains characters forbidden in nix store (like `~`)

Reply:
Thx. Fixed.